appveyor.yml

```diff
@@ -17,13 +17,17 @@ environment:
       TEST_FRAMEWORK: net452
       LIB_FRAMEWORK: net45
     - CONFIGURATION: Debug
-      CONSOLE_FRAMEWORK: netcoreapp1.0
-      LIB_FRAMEWORK: netstandard1.4
-      TEST_FRAMEWORK: netcoreapp1.0
```

Comment:
I was thinking if it makes sense to only configure the build+release jobs for the latest framework and release for older ones.